examples/gno.land/r/sys/params/params.gno

```diff
@@ -0,0 +1,37 @@
+package params
```

Comment:
Maybe we should add a few sentences about `sys/params`, since it took me a second to understand that this pipes into the Params Keeper we introduced a while back, and just enables usage of it through a GovDAO proposal

Reply:
6d99f19b7